packages/schema-generator/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@aws-amplify/schema-generator",
+  "version": "0.0.1-beta.1",
```

Comment:
very nit, such picky: [semver recommends](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) starting 0.1.0 rather than 0.0.1

Reply:
Updated to 0.1.0.